pkg/controllers/job/job_controller_handler.go

```diff
@@ -324,20 +324,36 @@ func (cc *Controller) deletePod(obj interface{}) {
 		return
 	}
 
-	req := apis.Request{
-		Namespace: pod.Namespace,
-		JobName:   jobName,
-		TaskName:  taskName,
-
-		Event:      bus.PodEvictedEvent,
-		JobVersion: int32(dVersion),
-	}
-
 	if err := cc.cache.DeletePod(pod); err != nil {
 		klog.Errorf("Failed to delete Pod <%s/%s>: %v in cache",
 			pod.Namespace, pod.Name, err)
 	}
 
+	if pod.Status.Phase == v1.PodSucceeded {
+		return
+	}
+
+	var req apis.Request
+	if pod.Status.Phase == v1.PodFailed && pod.Status.Reason == "Evicted" {
```

Comment:
remove this, we should treat every other phase as Evicted. This is the design intention.

Reply:
I mean only ignore `Succeeded` phase.

Reply:
And also see https://github.com/volcano-sh/volcano/issues/791#issuecomment-625581151